syscall/devices/hostresolver/hostresolver.c

```diff
@@ -102,7 +102,7 @@ static int _hostresolver_getaddrinfo(
                 &retval, node, service, hints, &handle) != OE_OK)
         {
             ret = OE_EAI_SYSTEM;
-            OE_RAISE_ERRNO(OE_EINVAL);
+            OE_RAISE_ERRNO(oe_errno);
```

Comment:
There may have been a reason to swallow the error in this file. Need to look into this.